src/test/users.test.js

```diff
@@ -140,4 +139,88 @@ describe('Users Related Tests', async () => {
     expect(res.status).to.be.equal(200);
     expect(res.body).to.have.property('message', 'Logged out successfully');
   });
+
+  it('should update a user', async() => {
+    const token = await siginIn(mockAdmin);
+
+    const res = await chai
+      .request(app)
+      .patch('/api/users/update/3')
+      .send(mockUpdate)
+      .set('auth', token);
+
+    expect(res.status).to.be.equal(200);
+    expect(res.body).to.have.property(
+      'message',
+      'User Updated',
+    );
+  });
+
+  it('should fiind Names not provided', async() => {
```

Comment:
let's have a good description of the tests. For example on this one you can say, it should throw an error when the names are not provided